ci/certification/dummy.yml

```diff
@@ -0,0 +1,1798 @@
+.certification:dummy:
+  stage: certification-dummy
+  extends: .certification-generic
+  variables:
+    DEVICE_UNDER_TEST: "prplmesh"
+  needs:
+    - job: build-in-docker
+
+MAP-4.2.1:dummy:
+  extends: .certification:dummy
+  rules:
+    # If the last commit description contains the test name and we're not on master, run it.
+    # Otherwise, make it a manual job
+    - if: '$CI_COMMIT_DESCRIPTION =~ /.*MAP-4.2.1:dummy.*/ && $CI_COMMIT_REF_NAME != "master"'
+      when: on_success
+    - when: manual
+      allow_failure: true
```

Comment:
It really sucks that this can't move to .certification-generic... But indeed, seems to be a limitation of the gitlab syntax.